Widescreen patch
Hoax - Daytime
Knight in right tower: adjust position, do not hide feet, prevent hiding based on camera offset
--- original
+++ modified
@@ -82,13 +82,12 @@
 gosub inl[:LABEL_0]
 consume
 LABEL_11:
-call 101, stor[0], 0x254, 0xfffffeae, 0xffffff98
+call 101, stor[0], 0x1fe, 0xfffffeae, 0xffffff98
 call 103, stor[0], 0x0, 0xf00, 0x0
 call 96, 0x4
-call 288, 0x5
-call 288, 0x6
 LABEL_12:
 yield
+jmp inl[:LABEL_12]
 call 257, stor[24], stor[25]
 jmp_cmp >, 0x5, stor[24], inl[:LABEL_13]
 call 106, 0x1
